runtime/android/core/src/org/xwalk/core/XWalkCoreWrapper.java

```diff
@@ -160,8 +186,8 @@ private boolean findSharedCore() {
             return false;
         }
 
-        Log.d(TAG, "Running on shared mode");
         application.addResource(mBridgeContext.getResources());
+        Log.d(TAG, "Running on shared mode");
```

Comment:
Since you're moving this line anyway, please use "in shared mode" instead of "on shared mode".